internal/status/statefulset_status.go

```diff
@@ -16,32 +16,21 @@ package status
 
 import (
 	"context"
+	"encoding/json"
 
-	appsv1 "k8s.io/api/apps/v1"
-	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/lib"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
 )
 
-var msgForReason = map[string]string{
-	lib.ObjectDeletionStartStatus:   "Started deleting objects",
-	lib.ObjectDeletionDoneStatus:    "Successfully deleted all objects",
-	lib.ObjectDeletionTimeoutStatus: "Error, timed out while deleting objects",
-}
-
-func msgFoundInStatus(conditions []appsv1.StatefulSetCondition, msg string) bool {
-	for _, c := range conditions {
-		if c.Type == lib.AKOConditionType && c.Message == msg {
-			return true
-		}
-	}
-	return false
-}
+const ObjectDeletionStatus = "ObjectDeletionStatus"
```

Comment:
Not too particular but shall we call it `AviObjectDeletionStatus` ?

Reply:
yeah, I'll change it